files/botautoresponse.json

```diff
@@ -787,6 +787,15 @@
                 ".feather"
             ]
         ],
-        "response": "https://hypixel.net/threads/feather-client-what-you-should-know-about-this-client.4801748/"
-    }
+        "response": "<https://hypixel.net/threads/feather-client-what-you-should-know-about-this-client.4801748/>"
+    },
+    {
+        "unclebot": true,
+        "triggers": [
+            [
+                ".java"
+            ]
+        ],
+        "response": "You should not need Java if you are using SkyClient however if required we recommend Adoptium.\n Make sure to set your architecture and operating system:\n<https://adoptium.net/temurin/releases/?version=8>"
```

Comment:
"you should not need java" is misinformation
also consider linking the essential guide, https://essential.gg/support/troubleshooting/install-java

Reply:
ive been constantly told you dont :pensive:

Reply:
You should not need to manually install Java with SkyClient, but if you need to we recommend Adoptium.\nFilter by your chip architecture and operating system at https://adoptium.net/temurin/releases/?version=8, and download the right version.

Reply:
Minecraft Java edition **requires** java to run, it's coded in Java, just because it comes with the mc launcher so you don't usually need to manually install it doesn't mean that you don't need Java for SkyClient